api/cmd/managerapi.go

```diff
@@ -99,7 +108,9 @@ func NewManagerAPICommand() *cobra.Command {
 			}
 			if err := store.InitStores(); err != nil {
 				log.Errorf("init stores fail: %w", err)
-				panic(err)
+				fmt.Printf("%s\n", err)
```

Comment:
We should output exceptions to stderr, not stdout.

Reply:
Oops!😅😂